lib/activity/router.js

```diff
@@ -80,7 +80,7 @@ module.exports = function route(callback) {
           await callback(context, subscription, slack);
         } catch (err) {
           if (isPermanentError(err)) {
-            context.log.info({ err }, 'Permanent error from Slack. Removing subscription');
+            context.log.info({ err, subscription, payload: context.payload }, 'Permanent error from Slack. Removing subscription');
```

Comment:
I think `context.payload` might be a bit data heavy here, should we only log the parts out of that we care about like the repo name with owner?